airflow/www/templates/airflow/ti_code.html

```diff
@@ -21,10 +21,10 @@
 {% block title %}DAGs - {{ appbuilder.app_name }}{% endblock %}
 
 {% block content %}
-  {{ super() }}
-  <h4>{{ title }}</h4>
-  {% for k, v in html_dict.items() %}
-    <h5>{{ k }}</h5>
-    {{ v }}
-  {% endfor %}
+{{ super() }}
+<h4>{{ title }}</h4>
+{% for k, v in html_dict.items() %}
+<h5><strong>{{ k }}</strong></h5>
```

Comment:
We should probably add CSS to style `h5` instead of adding a `strong`? Or maybe this does not really matter at this point. cc @bbovenzi